StrictEmit/StrictEmit.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>netstandard2.0;net4</TargetFrameworks>
```

Comment:
Can a lower version of .NET be targeted while we're at it?

Reply:
.Net Framework: yes, we can go lower.
.Net Standard: needs code changes since .Net Standard 1.x has `Type.GetTypeInfo().IsValueType` instead of `Type.IsValueType`. This can be done with `#if`'s or an utility method.